test/dogma/rule/taken_name_test.exs

```diff
@@ -0,0 +1,69 @@
+defmodule Dogma.Rule.TakenNameTest do
+  use RuleCase, for: TakenName
+  use ShouldI
+
+  defp lint(script) do
+    script |> Script.parse!( "" ) |> fn s -> TakenName.test(@rule, s) end.()
```

Comment:
Rules should be dispatched to using the protocol, not by invoking the module function directly.